pkg/scheduler/core/scheduling_queue_test.go

```diff
@@ -241,12 +250,20 @@ func TestPriorityQueue_Delete(t *testing.T) {
 }
 
 func TestPriorityQueue_MoveAllToActiveQueue(t *testing.T) {
-	q := NewPriorityQueue()
+	clock := fakeClock{}
+	q := NewPriorityQueueWithClock(nil, &clock)
 	q.Add(&medPriorityPod)
 	q.unschedulableQ.Add(&unschedulablePod)
 	q.unschedulableQ.Add(&highPriorityPod)
 	q.MoveAllToActiveQueue()
-	if q.activeQ.data.Len() != 3 {
+
+	if q.activeQ.Len() != 1 {
+		t.Error("Expected all unschedulable items to be in backoff.")
+	}
+	clock.t = clock.t.Add(2 * time.Second)
+	q.moveNextBackoffToActive()
```

Comment:
why are we doing this twice?

Reply:
Maybe I should add a comment about this. We are adding two pods and moveNextBackoffToActive only moves one pod to the activeQ at a time.